vagrant/Vagrantfile

```diff
@@ -86,8 +80,83 @@ Vagrant.configure("2") do |config|
     node.vm.network "forwarded_port", type: "dhcp", guest: 8858, host: 8858, auto_correct: true
     node.vm.network "forwarded_port", type: "dhcp", guest: 8860, host: 8859, auto_correct: true
     node.vm.synced_folder "../", "/vagrant-root"
-    node.vm.provision :shell, path: "provisioning/provision_node.sh", args: args
     node.vm.boot_timeout = 600
   end
+
+# Provision each VM
+  config.vm.provision :ansible do |ansible|
+    require 'json'
+
+    packages_to_install = [{
+      "package_name" => "nmoscommon",
+      "repo_url" => "https://github.com/bbc/nmos-common.git",
+      "branch" => repo_branches[0]
+    },
+    {
+      "package_name" => "nmosreverseproxy",
+      "repo_url" => "https://github.com/bbc/nmos-reverse-proxy.git",
+      "branch" => repo_branches[2],
+      "service_file" => "ips-reverseproxy-common.service",
+      "reverse_proxy_file" => "ips-reverseproxy-common.conf"
+    },
+    {
+      "package_name" => "mdnsbridge",
+      "repo_url" => "https://github.com/bbc/nmos-mdns-bridge.git",
+      "branch" => repo_branches[1],
+      "service_file" => "python-mdnsbridge.service",
+      "reverse_proxy_file" => "ips-api-mdnsbridge.conf"
+    }]
+
+    regquery_packages_to_install = packages_to_install + [{
+      "package_name" => "registryaggregator",
+      "repo_url" => "https://github.com/bbc/nmos-registration.git",
+      "branch" => repo_branches[5],
+      "service_file" => "python-registryaggregator.service",
+      "reverse_proxy_file" => "ips-api-registration.conf"
+    },
+    {
+      "package_name" => "registryquery",
+      "repo_url" => "https://github.com/bbc/nmos-query.git",
+      "branch" => repo_branches[4],
+      "service_file" => "python-registryquery.service",
+      "reverse_proxy_file" => "ips-api-query.conf"
+    }]
+
+    auth_packages_to_install = packages_to_install + [{
+      "package_name" => "nmos-auth",
+      "repo_url" => "https://github.com/bbc/nmos-auth-server.git",
+      "branch" => repo_branches[7],
+      "service_file" => "python-nmos-auth.service",
+      "reverse_proxy_file" => "ips-api-auth.conf"
+    }]
+
+    node_packages_to_install = packages_to_install + [{
```

Comment:
Does this not require nmoscommon/reverseproxy and mdnsbridge? Or are they installed via other means?

Auth server probably needs the first two as well.

Reply:
There is a base list of packages that all VMs use which is concatenated with the VM specific one
`= packages_to_install + [{`

Reply:
I see, didn't look closely enough!